src/inspect_ai/_cli/log.py

```diff
@@ -164,9 +183,11 @@ def dump_command(path: str, header_only: bool, resolve_attachments: bool) -> Non
 )
 @click.option(
     "--resolve-attachments",
-    type=bool,
-    is_flag=True,
-    default=False,
+    type=click.Choice(["full", "core"]),
+    flag_value="core",
+    is_flag=False,
+    default=None,
```

Comment:
I notice that these args are different than [here](https://github.com/UKGovernmentBEIS/inspect_ai/pull/2552/files#diff-18829141d4859bd887124c0c522e91bf1046f3fc79241849f0af6d1074d73d9cR149-R150)